portal-web/test/functional/com/liferay/portalweb/paths/plugin/workflow/WorkflowMetrics.path

```diff
@@ -234,6 +239,36 @@
 	<td>//td[contains(@class,'table-title')]//span[contains(text(),'${key_assigneeName}')]/../../following-sibling::td[contains(@data-testid,'taskCount')]//a[contains(text(),'${key_taskCount}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>METRICS_WORKLOAD_BY_ASSIGNEE_TASKCOUNT_OVERDUE</td>
+	<td>xpath=(//span[contains(text(),'${assigneeName}')]/ancestor::td[contains(@class,'table-title')]/following-sibling::td//a)[1]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>METRICS_WORKLOAD_BY_ASSIGNEE_TASKCOUNT_ON_TIME</td>
+	<td>xpath=(//span[contains(text(),'${assigneeName}')]/ancestor::td[contains(@class,'table-title')]/following-sibling::td//a)[2]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>METRICS_WORKLOAD_BY_ASSIGNEE_TASKCOUNT_TOTAL_PENDING</td>
+	<td>xpath=(//span[contains(text(),'${assigneeName}')]/ancestor::td[contains(@class,'table-title')]/following-sibling::td//a)[1]</td>
+	<td></td>
+</tr>
```

Comment:
Hey @calvinctvo ,
This path is the same value as METRICS_WORKLOAD_BY_ASSIGNEE_TASKCOUNT_OVERDUE.

Would you please take a look?